tools/clang/test/HLSLFileCheck/hlsl/types/struct/bitfields_error.hlsl

```diff
@@ -1,6 +1,70 @@
-// RUN: %dxc /T vs_6_0 /E main %s | FileCheck %s
+// RUN: %dxc /T vs_6_2 /E main %s | FileCheck %s
```

Comment:
This file seems more suitable as a VerifierTest.

Reply:
The replacement PR will make it a VerifierTest.